pkgs/applications/office/paperless-ngx/default.nix

```diff
@@ -34,11 +38,11 @@ let
 in
 py.pkgs.pythonPackages.buildPythonApplication rec {
   pname = "paperless-ngx";
-  version = "1.6.0";
+  version = "1.7.1";
 
   src = fetchurl {
```

Comment:
I'd suggest adding a comment here, why we don't use fetchFromGithub (e.g. frontend already built and included)

Reply:
Done.